### PR TITLE
Support building OCI images with apko.

### PR DESCRIPTION
**Pull request by kaniini**
Demo here: https://github.com/kaniini/apko-test

### CODE REVIEW

File: apko-build/action.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  repository_owner:
+    description: |
+      The repository owner's GitHub username.
+    default: ${{ github.repository_owner }}
+
+  token:
+    description: |
+      The repository owner's GitHub token.
+    default: ${{ github.token }}
```
Comment (mattmoor) [inline apko-build/action.yaml:28]: I think if we just tell folks that have to have auth configured, and that we respect the Docker keyring by default we can kill this and have folks use the login plugin.

File: apko-build/action.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+runs:
+  using: docker
+  image: 'docker://ghcr.io/chainguard-dev/apko:v0.1'
```
Comment (mattmoor) [inline apko-build/action.yaml:32]: ... or I'm guessing the container this runs in creates trouble for the above, which is what you were saying in slack 🤔

**Review by mattmoor — APPROVED** [review]
I'm guessing I answered my own question here.

File: apko-build/action.yaml (hunk 2 above)
Comment (kaniini) [reply apko-build/action.yaml:32]: I'm not sure, I can try the docker login action, but I'm pretty sure it wasn't being copied into the container.

File: apko-build/action.yaml (hunk 1 above)
Comment (kaniini) [reply apko-build/action.yaml:28]: Doesn't work, because the docker credentials don't get passed through to the apko image, unfortunately...

**Pull request merged**